stack.yaml

```diff
@@ -1,4 +1,12 @@
-resolver: lts-9.17
+resolver: lts-11.9
 packages:
 - .
 - example-project
+extra-deps:
```

Comment:
This is a library and it cannot rely on `extra-deps` like this. I hope Amazonka makes its way back into LTS soon, but for now I'd stay with LTS 10.